.github/workflows/_btp-integration-tests.yaml

```diff
@@ -0,0 +1,81 @@
+name: btp integration tests
+
+on:
+  workflow_call:
+
+jobs:
+  btp-integration-tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: tests/btp/hack
+    steps:
+      - uses: actions/checkout@v4
+      - uses: opentofu/setup-opentofu@v1
```

Comment:
this will not work in kyma-project org as it is not whitelisted action.
If you are running on ubuntu agent, then nothing needs to be installed.
terraform CLI is installed by default.
just replace `tofu`->`terraform`

Reply:
Thanks for that, changed as suggested